docker/api/exec_api.py

```diff
@@ -137,7 +137,7 @@ def exec_start(self, exec_id, detach=False, tty=False, stream=False,
             (generator or str or tuple): If ``stream=True``, a generator
             yielding response chunks. If ``socket=True``, a socket object for
             the connection. A string containing response data otherwise. If
-            ``demux=True``, stdout and stderr are separated.
+            ``demux=True``, a tuple of two bytes items: stdout and stderr.
```

Comment:
Sure, that was exactly what I wrote originally. But since we have to follow PEP8 standard, if written in this way, a new line is needed for the last word, like this:
![image](https://user-images.githubusercontent.com/12481493/54536686-1ff7cb80-49cc-11e9-917f-4047224b1473.png)

Personally, I thought it made the code not that pretty, so I changed this sentence into a shorter expression with the same meaning. But it turns out that the new expression is more ambiguous, if  the situation in the screenshot above doesn't matter for you, I am glade to change the expression into this way.

Reply:
No problem with the new line. The content is more important

Reply:
Okay! I will change it